lib/product-import.coffee

```diff
@@ -90,14 +90,11 @@ class ProductImport
     if @filterUnknownAttributes then @_summary.unknownAttributeNames = []
 
   summaryReport: (filename) ->
-    if @_summary.created is 0 and @_summary.updated is 0 and @_summary.failed is 0
-      message = 'Summary: nothing to do, everything is fine'
-    else
-      message = "Summary: there were #{@_summary.created + @_summary.updated} imported products " +
-        "(#{@_summary.created} were new and #{@_summary.updated} were updates)"
+    message = "Summary: there were #{@_summary.created + @_summary.updated} imported products " +
+      "(#{@_summary.created} were new and #{@_summary.updated} were updates)."
 
-    if @_summary.emptySKU > 0
-      message += "\nFound #{@_summary.emptySKU} empty SKUs from file input"
+    if @_summary.missingSKU > 0
+      message += "\nFound #{@_summary.missingSKU} products which do not have SKU and won't be imported."
```

Comment:
It filters out products which do not have at least one variant with SKU so "products".